Graphics/ShaderTools/src/ShaderToolsCommon.cpp

```diff
@@ -359,15 +399,24 @@ static std::string UnrollShaderIncludesImpl(ShaderCreateInfo ShaderCI, std::unor
         }
 
         PrevIncludeEnd = IncludeEnd;
-    });
+    };
+
+    auto ErrorCallback = [&](const std::string& Error) //
+    {
+        LOG_ERROR("Failed to parse file '", ShaderCI.FilePath, "' ", Error);
```

Comment:
If error happens within one of the include files, line and position information will be useless as it will not be clear in which file exactly the error happened.